wagtail/contrib/frontend_cache/backends.py

```diff
@@ -293,7 +293,7 @@ def _get_subscription_id(self):
 
     def _get_client_kwargs(self):
         return {
-            "credentials": self._get_credentials(),
+            "credential": self._get_credentials(),
```

Comment:
We'll need to consider how this change affects end-users. Presumably this code has been incompatible with azure-mgmt-cdn >=10 up to this point, and this change will break things for a user who has the dependency pinned to <10. At the very least we need to document the version requirements in the docs and release notes - ideally we'd identify the version in use and pass the appropriate kwarg.

Reply:
Good point, I’ll add a version check to make sure the change is backwards-compatible. Should I also add a deprecation warning, so we can remove the compatibility code in a future release?

Reply:
@sephii Yes please!

Reply:
Here you go! Let me know if it matches what you had in mind (and feel free to add any mention about the change in the release notes).